.idea/.name

```diff
@@ -1 +1 @@
-DotsenkoRSSReader2
\ No newline at end of file
+RSSReader
```

Comment:
Also, you highly need to setup .gitignore and remove this .idea/ files from git.